.travis.yml

```diff
@@ -10,11 +14,13 @@ install:
   # (The conda installation steps below are taken from http://conda.pydata.org/docs/travis.html)
   # We do this conditionally because it saves us some downloading if the
   # version is the same.
+  - find .
```

Comment:
Do we need this or is it just for debugging?

Reply:
yes good catch, I'll remove it